app/views/admin/settings/forms/_meta_keywords.html.erb

```diff
@@ -1,4 +1,6 @@
-<%= form_for(Settings::General.new, url: admin_settings_general_settings_path) do |f| %>
+<%= form_for(Settings::General.new,
+             url: admin_settings_general_settings_path,
+             html: { data: { action: "submit->config#activateLightConfirmation" } }) do |f| %>
```

Comment:
I love how straightforward this sync up for the light confirm is - so reusable and clear 😭